compiler/rustc_ast/src/ast.rs

```diff
@@ -2585,7 +2576,7 @@ pub struct Visibility {
 #[derive(Clone, Encodable, Decodable, Debug)]
 pub enum VisibilityKind {
     Public,
-    Crate(CrateSugar),
+    Crate,
     Restricted { path: P<Path>, id: NodeId },
```

Comment:
`Crate` can be merged into `Restricted`, it's a special case of it.

(Also span of the `crate` will be properly kept in that case, which can make difference with macro 2.0 hygiene.)

Reply:
Done.